projects/ui-components/lf-login/lf-login.component.ts

```diff
@@ -22,7 +22,7 @@ export class LfLoginComponent implements OnChanges, OnDestroy {
     return this.mode === LoginMode.Menu;
   };
   // TODO localize
-  @Input() sign_in_text: string = 'Sign in to Laserfiche';
+  @Input() sign_in_text: string = 'Sign in';
```

Comment:
can we use the localization service and translate the ones that do have translations?

Reply:
I guess it doesn't matter either way right now